_1327/information_pages/tests.py

```diff
@@ -201,6 +202,22 @@ def test_save_version(self):
 		self.assertEqual(versions[0].revision.comment, 'hallo Bibi Blocksberg')
 		self.assertEqual(versions[1].revision.comment, 'test version')
 
+	def test_last_author(self):
+		# create a second user
+		user2 = mommy.make(UserProfile, is_superuser=True)
+		user2.groups.add(self.group)
+
+		# create second revision
+		self.document.text = 'christi was here'
+		with transaction.atomic(), revisions.create_revision():
+			self.document.save()
+			revisions.set_user(user2)
+			revisions.set_comment('test version 2')
+
+		# test whether last author is part of page
+		response = self.app.get(self.document.get_view_url(), user=self.user)
+		self.assertIn('<i>by</i> {}'.format(user2.username), response.body.decode('utf-8'))
```

Comment:
It would make sense to first test whether `self.user` was the last author before `user2` created a new revision, so please add a request to the top of this test, where you check that `self.user` is displayed as the author of the document first